test/cluster.sh

```diff
@@ -55,18 +58,29 @@ function install_knative_serving() {
     build_knative_from_source
     INSTALL_ISTIO_CRD_YAML="${ISTIO_CRD_YAML}"
     INSTALL_ISTIO_YAML="${ISTIO_YAML}"
-    # TODO(#2122): Use RELEASE_YAML once we have monitoring e2e.
-    INSTALL_RELEASE_YAML="${RELEASE_NO_MON_YAML}"
+    # TODO(#2122): Install monitoring as well once we have e2e testing for it.
+    INSTALL_RELEASE_YAML="${SERVING_YAML}"
   fi
+  # TODO: Should we install build from a nightly release?
+  # This bit of jq extracts the asset named "release.yaml" from the
```

Comment:
I would prefer a simpler solution: get it from GCS, `latest` dir.

Reply:
And now another concern: you're hardcoding the latest public build release here. Any changes to HEAD that need a newer build will fail (even though the changes will be available in the vendored build dir, for example).

Reply:
Do we want to allow HEAD from serving to depend on unreleased features from Build? Doing this will introduce more trickiness in upgrade scenarios, if (for example) Build has to be upgraded before serving can be upgraded.

Using the latest published release was intentional, but I can switch this to a nightly release somewhere if you prefer.

Reply:
I confess I'm not sure what's the best approach here, I can see advantages and disadvantages in all of them. :P I'll defer the choice of public/nightly/vendor to @mattmoor as he might foresee issues I can't.

But I still prefer a yaml fetcher that uses a simpler approach through a simple fetch without `jq` (bonus: one less tool to depend on).

Reply:
Switched to https://storage.googleapis.com/knative-releases/build/latest/release.yaml as a static file, rather than using the GitHub API to find the YAML file.